src/include/storage/sql_table.h

```diff
@@ -11,6 +11,12 @@
 #include "storage/write_ahead_log/log_record.h"
 #include "transaction/transaction_context.h"
 
+namespace terrier {
+// Forward Declaration
+class LargeSqlTableTestObject;
+class RandomSqlTableTransaction;
+}  // namespace terrier
```

Comment:
I don't think thats actually needed after the addition of the `Layout()` method. I've removed it.

Reply:
Marking this one resolved, but it may come back from addressing a comment on the in-depth review. (https://github.com/cmu-db/terrier/pull/459#discussion_r313876184)